### PR TITLE
Series of smallish changes - triggered by translations.

### PR DESCRIPTION
**Pull request by smoe**

### CODE REVIEW

**Comment by hansu** [comment]
Will you not rather resolve the merge conflicts for #1926 than creating new ones?

**Comment by smoe** [comment]
> Will you not rather resolve the merge conflicts for #1926 than creating new ones?

The translations work on every machine whenever I have a spare minute. The changes to the English depend on the machine I am working on.

**Comment by hansu** [comment]
But you can check out every branch on every machine. I don't see the problem...

**Comment by smoe** [comment]
I admit that these conflicts get on my nerves, too, so I should consider to adjust my routine.

**Comment by petterreinholdtsen** [comment]
I had a look at this, and realize I do not know the asciidoc format enough to understand the effect of all the formatting changes.

**Comment by hansu** [comment]
@smo just to inform you about your homemade conflicts ;-)

File: docs/src/gui/ngcgui.adoc [hunk 1]
```diff
@@ -24,7 +24,11 @@ image::images/ngcgui.png[align="center"]
 * 'PYNGCGUI' is an alternate, python implementation of ngcgui.
 * 'PYNGCGUI' can run as a standalone application or can be embedded as
   a tab page (with its own set of multiple subroutine tabs) in any
+<<<<<<< HEAD
```
Comment (hansu) [inline docs/src/gui/ngcgui.adoc:27]: ooops :wink: 


and some further in this file

File: docs/src/gui/panelui.adoc [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 == Introduction
 
+<<<<<<< HEAD
```
Comment (hansu) [inline docs/src/gui/panelui.adoc:13]: in this file as well

File: docs/src/gui/panelui.adoc [hunk 3]
```diff
@@ -18,6 +19,16 @@ Panelui is configurable using an INI style text file to define button
...
+=======
+Panelui is a userspace component to interface buttons to LinuxCNC or HAL:
+* It decodes MESA 7I73 style key-scan codes and calls the appropriate routine.
+* It gets input from a realtime component - sampler.
+  Sampler gets it's input from either the MESA 7i73 or sim_matrix_kb component.
+* Panelui is configurable using an INI style text file to define button
+  types, HAL pin types, and/or commands.
+* It can be extended using a Python based 'handler' file to add functions.
+
+>>>>>>> 2bd0b2e818 (More changes, many python -> Python)
```
Comment (hansu) [inline docs/src/gui/panelui.adoc:31]: this is duplicated

File: docs/src/gui/qtvcp-widgets.adoc [hunk 4]
```diff
@@ -378,26 +378,38 @@ ActionButton #action_aux{
...
-==== Call python commands on state
+==== Call Python commands on state
 
-The python_command_option allow small snippets of python code to be run
+The python_command_option allow small snippets of Python code to be run
 from the push of a button, with out having to edit the handler file.
 (though it can call functions in the handler file)
 When using the command_string properties.
 
+<<<<<<< HEAD
```
Comment (hansu) [inline docs/src/gui/qtvcp-widgets.adoc:388]: here as well

File: docs/src/gui/qtvcp-widgets.adoc [hunk 5]
```diff
@@ -378,26 +378,38 @@ ActionButton #action_aux{
...
 * 'true_python_cmd_string' - a python command that will be called when
   the button is toggled true
 * 'false_python_cmd_string' - a python command that will be called when
+=======
+* 'true_python_cmd_string' - a Python command that will be called when
+  the button is toggled true 
+* 'false_python_cmd_string' - a Python command that will be called when
+>>>>>>> 2bd0b2e818 (More changes, many python -> Python)
```
Comment (hansu) [inline docs/src/gui/qtvcp-widgets.adoc:396]: duplicated

File: docs/src/gui/qtvcp-widgets.adoc [hunk 6]
```diff
@@ -2037,7 +2049,7 @@ available.
 
 ==== Properties
 
-OriginOfsetView has properties that can be set in designer, in python
+OriginOfsetView has properties that can be set in designer, in Python
```
Comment (hansu) [inline docs/src/gui/qtvcp-widgets.adoc:2052]:
```suggestion
OriginOffsetView has properties that can be set in designer, in Python
```

File: docs/src/gui/qtvcp-widgets.adoc [hunk 7]
```diff
@@ -2395,7 +2407,7 @@ It is based on PyQt's QDialog.
 
 This is a dialog to display an edit line for information entry, such as
 origin offset.
-It returns the entry via STATUS messages using a python DICT.
+It returns the entry via STATUS messages using a Python DICT.
```
Comment (hansu) [inline docs/src/gui/qtvcp-widgets.adoc:2410]:
```suggestion
It returns the entry via STATUS messages using a Python dict.
```

File: docs/src/gui/qtvcp-widgets.adoc [hunk 8]
```diff
@@ -2410,7 +2422,7 @@ image::images/qtvcp_calculator.png["QtVCP Calculator",scale="25%"]
 
 This is a dialog to display a calculator for numeric entry, such as
 origin offset.
-It returns the entry via STATUS messages using a python DICT.
+It returns the entry via STATUS messages using a Python DICT.
```
Comment (hansu) [inline docs/src/gui/qtvcp-widgets.adoc:2425]:
```suggestion
It returns the entry via STATUS messages using a Python dict.
```

**Review by hansu — CHANGES_REQUESTED** [review]

**Comment by hansu** [comment]
Nice work and thanks for immediately rebasing this :slightly_smiling_face:

File: docs/src/gui/panelui.adoc (hunk 2 above)
Comment (smoe) [reply docs/src/gui/panelui.adoc:13]: Done.

File: docs/src/gui/qtvcp-widgets.adoc (hunk 4 above)
Comment (smoe) [reply docs/src/gui/qtvcp-widgets.adoc]: done.

File: docs/src/gui/panelui.adoc (hunk 3 above)
Comment (smoe) [reply docs/src/gui/panelui.adoc]: done

**Comment by smoe** [comment]
Let me squash these.

File: docs/src/gui/ngcgui.adoc [hunk 9]
```diff
@@ -336,7 +336,7 @@ be installed independently and located in the user PATH.
 === Embedding PYNGCGUI as a GladeVCP tab page in a GUI
 
 The following INI file items go in the [DISPLAY] section for use with the
-AXIS, gscreen, or Touchy GUIs.  (See additional sections below for additional
+AXIS, GScreen, or Touchy GUIs.  (See additional sections below for additional
```
Comment (hansu) [inline docs/src/gui/ngcgui.adoc:339]:
```suggestion
AXIS, gscreen, or Touchy GUIs.  (See additional sections below for additional
```

File: docs/src/gui/ngcgui.adoc [hunk 10]
```diff
@@ -55,7 +55,7 @@ A number of demonstration configurations are located in the sim directory
 of the Sample Configurations offered by the LinuxCNC configuration picker. The
 configuration picker is on the system's main menu: Applications > CNC > LinuxCNC
 
-Examples are included for the AXIS, Touchy, gscreen, and GMOCCAPY.
+Examples are included for the AXIS, Touchy, GScreen, and GMOCCAPY.
```
Comment (hansu) [inline docs/src/gui/ngcgui.adoc:58]:
```suggestion
Examples are included for the AXIS, Touchy, gscreen, and GMOCCAPY.
```

File: docs/src/gui/ngcgui.adoc [hunk 11]
```diff
@@ -24,8 +24,8 @@ image::images/ngcgui.png[align="center"]
 * 'PYNGCGUI' is an alternate, python implementation of ngcgui.
 * 'PYNGCGUI' can run as a standalone application or can be embedded as
   a tab page (with its own set of multiple subroutine tabs) in any
-  GUI that supports embedding of GladeVCP applications AXIS, Touchy, gscreen
-  and gmoccapy.
+  GUI that supports embedding of GladeVCP applications AXIS, Touchy, GScreen
```
Comment (hansu) [inline docs/src/gui/ngcgui.adoc:27]:
```suggestion
  GUI that supports embedding of GladeVCP applications AXIS, Touchy, gscreen
```

**Review by hansu — APPROVED** [review]

**Comment by hansu** [comment]
> Let me squash these.

Sure, it's your branch :wink:

**Comment by smoe** [comment]
Argh. @hansu It seems like I have accidentially overwritten the most commits of yours when I force-uploaded my local rebase. From what I see in my inbox, this was about how Gscreen should be spelled. This is highly inconsistent, throughout LinuxCNC documentation and apps :o/ 

$ grep -r gscreen ../docs/|grep -v pot|grep -v po | wc
grep: ../docs/man/index.db: binary file matches
    397    6672   62334
$ grep -r Gscreen ../docs/|grep -v pot|grep -v po | wc
    582   11696   94301

**Comment by hansu** [comment]
> Argh. @hansu It seems like I have accidentially overwritten the most commits of yours when I force-uploaded my local rebase. From what I see in my inbox, this was about how Gscreen should be spelled. This is highly inconsistent, throughout LinuxCNC documentation and apps :o/
> 
> $ grep -r gscreen ../docs/|grep -v pot|grep -v po | wc grep: ../docs/man/index.db: binary file matches 397 6672 62334 $ grep -r Gscreen ../docs/|grep -v pot|grep -v po | wc 582 11696 94301

No worries, I still have it locally.

Yes the spelling of gscreen should be unified. But GScreen at least not ;-).

**Comment by hansu** [comment]
But your commit message doesn't fit any longer :laughing:

**Pull request merged**